cypress/integration/analyticsFlows/analyticsNavigation.spec.js

```diff
@@ -43,4 +43,34 @@ describe('Analytics navigation', () => {
       .findByRole('button', { name: 'Infinity' })
       .should('have.attr', 'aria-current', 'page');
   });
+
+  it('should hide organizations menu', () => {
+    cy.findByRole('navigation', { name: 'Organizations menu' }).should(
+      'not.exist',
+    );
+  });
+});
+
+describe('when user is admin of an organization', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/dashboard/analytics');
+      });
+    });
+  });
+
+  it('should show organizations menu', () => {
+    cy.findByRole('navigation', { name: 'Dashboards' }).should('exist');
+  });
+
+  it('should navigate to correct organization analytics dashboard', () => {
+    cy.findByText('Bachmanity Analytics Dashboard').as('organizationLink');
+    cy.get('@organizationLink').click();
```

Comment:
```suggestion
    cy.findByText('Bachmanity Analytics Dashboard').click();
```

suggestiong (not-blocking) - since we only use `organizationLink` once we could just inline it 🙂

Reply:
I tried this in my code and my tests broke. I wasn't sure if it had to do with cypress syntax or something else, but it must've been something unrelated then 🤔 Happy to commit this!